src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -265,12 +271,43 @@
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
       <PkgDefEntry>BindingRedirect</PkgDefEntry>
     </ProjectReference>
-    <ProjectReference Include="..\VisualStudioInteractiveComponents\Roslyn.VisualStudio.InteractiveComponents.csproj">
-      <Name>InteractiveComponents</Name>
-      <!-- Do not include Roslyn.VisualStudio.InteractiveComponents.dll in the VSIX -->
+    <ProjectReference Include="..\..\Interactive\HostProcess\InteractiveHost64.csproj">
```

Comment:
The VSSDK is running MSBuild task on these references because they have `IncludeOutputGroupsInVSIX` metadata. The VSSDK targets are doing something similar to what I'm doing in the unit test project.